ideascube/models.py

```diff
@@ -311,6 +311,13 @@ def get_prep_value(self, value):
         return json.dumps(value)
 
 
+class LanguageField(models.CharField):
+    def __init__(self, *args, **kwargs):
+        kwargs['choices'] = get_all_languages()
+
```

Comment:
Why not?

I guess I just prefer separating my own processing from the parent « normal » stuff, feels a bit more readable...

In general, I tend to be very generous with empty lines, as I do feel that makes the code more readable. A bit like the pragraphs in this response, it's all about grouping your prose/code into semantic units. :wink: